apps/music/test/marionette/metadata_test.js

```diff
@@ -0,0 +1,109 @@
+/* global require, marionette, setup, suite, test */
+'use strict';
+
+var assert = require('assert');
+var Music = require('./lib/music.js');
+var ListviewHelper = require('./lib/listviewhelper.js');
+
+marionette('Music player playlist', function() {
+  var apps = {};
+
+  var client = marionette.client({
+    prefs: {
+      'device.storage.enabled': true,
+      'device.storage.testing': true,
+      'device.storage.prompt.testing': true
+    },
+
+    settings: {
+      'lockscreen.enabled': false,
+      'ftu.manifestURL': null
+    },
+
+    apps: apps
+  });
+
+  var music;
+
+  setup(function() {
+    music = new Music(client);
+
+  });
+
+  suite('Album art display', function () {
+    setup(function() {
+      client.fileManager.removeAllFiles();
+      client.fileManager.add([
+        {
+          // blue art.
+          type: 'music',
+          filePath: 'apps/music/test-data/playlists/a.ogg'
+        },
+        {
+          // red art
+          type: 'music',
+          filePath: 'apps/music/test-data/playlists/01.ogg'
+        },
+        {
+          // no art
+          type: 'music',
+          filePath: 'test_media/samples/Music/b2g.ogg'
+        },
+      ]);
```

Comment:
There are a bunch of undocumented assumptions in this fileManager.add() call. In order for the test to work, I think that the album, artist and song names for all three files have to be in a certain alphabetical order. Since these test files are shared with other tests, their metadata might change. So I think it would be good to add a comment documenting the requirements for these files, and maybe also specifying the title, album and artist for each of the files.